src/tools/interop/idt/capture/ecosystem/play_services_user/play_services_user.py

```diff
@@ -0,0 +1,66 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import os
+
+from capture.base import EcosystemCapture, UnsupportedCapturePlatformException
+from capture.file_utils import create_standard_log_name, print_and_write
+from capture.platform.android.android import Android
+
+
+class PlayServicesUser(EcosystemCapture):
```

Comment:
The main script entry-point might need to be able to query what Capture Capabilities are supportable. E.g. detect if the phone is rooted, etc.. Check if it is capable of play services user vs play services.

Reply:
I was hoping we could scope checks to their respective platform/ecosystem. For instance, if the current PlayServices impl needed to do some special check to function, that check should happen in the ecosystem init / start, and the entry-point of the script should eventually do error handling / better concurrency management for each ecosystem. How do you feel about this idea?